internal/ingress/controller/config/config.go

```diff
@@ -486,6 +486,10 @@ type Configuration struct {
 	SyslogHost string `json:"syslog-host"`
 	// SyslogPort port
 	SyslogPort int `json:"syslog-port",omitempty`
+
+	// NoTLSRedirectLocations is a "\n -" seperated list of locations
```

Comment:
please don't do this and just use a comma separated string like other configurations present in the configmap.